### PR TITLE
Simple public homefeed query and mutation

### PR DESCRIPTION
**Pull request by zeeshanakram3**
addresses #303

### CODE REVIEW

File: db/viewDefinitions.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ function getViewDefinitions(db) {
     video: [
       `is_excluded='0'`,
       `is_censored='0'`,
+      `is_public='1'`,
```
Comment (WRadoslaw) [inline db/viewDefinitions.js:14]: I think Atlas filters non public videos by itself in public feeds, but allows to see non public videos in Studio or directly through video URL

File: src/server-extension/resolvers/VideosResolver/index.ts [hunk 2]
```diff
@@ -194,6 +198,61 @@ export class VideosResolver {
...
+  @Mutation(() => SetPublicFeedResult)
+  @UseMiddleware(OperatorOnly(OperatorPermission.SET_PUBLIC_FEED_VIDEOS))
+  async setPublicFeedVideos(@Args() { videoIds }: SetPublicFeedArgs): Promise<SetPublicFeedResult> {
+    const em = await this.em()
+
+    return withHiddenEntities(em, async () => {
+      const result = await em
+        .createQueryBuilder()
+        .update<Video>(Video)
+        .set({ includeInHomeFeed: true })
```
Comment (WRadoslaw) [inline src/server-extension/resolvers/VideosResolver/index.ts:246]: How to remove video from home feed?

Comment (zeeshanakram3) [reply src/server-extension/resolvers/VideosResolver/index.ts]: Addressed in [4e77fc0](https://github.com/Joystream/orion/pull/304/commits/4e77fc062df80011ee1ac71885943701a405f5ee)

File: src/server-extension/resolvers/VideosResolver/index.ts [hunk 3]
```diff
@@ -234,16 +234,18 @@ export class VideosResolver {
...
+    @Args() { videoIds, operation }: SetOrUnsetPublicFeedArgs
+  ): Promise<SetOrUnsetPublicFeedResult> {
     const em = await this.em()
 
     return withHiddenEntities(em, async () => {
       const result = await em
         .createQueryBuilder()
         .update<Video>(Video)
-        .set({ includeInHomeFeed: true })
+        .set({ includeInHomeFeed: operation === 'set' })
```
Comment (WRadoslaw) [inline src/server-extension/resolvers/VideosResolver/index.ts:248]:
```suggestion
        .set({ includeInHomeFeed: operation === PublicFeedOperationType.SET })
```

**Review by WRadoslaw — APPROVED** [review]

**Pull request merged**